### PR TITLE
Docs review Query hit count accuracy

### PR DESCRIPTION
**Pull request by dvagnero**
Docs review of 
https://issues.redhat.com/browse/ISPN-14195
https://issues.redhat.com/browse/ISPN-14203
needs backport to 14

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/ref_query_ickle.adoc [hunk 1]
```diff
@@ -48,24 +48,26 @@ You can change the default limit by setting the `query.default-max-results` cach
...
-
-In this case `.isExact()` method or the `hit_count_exact` JSON field will be `false`.
-Setting this value to `Integer.MAX` would return accurate results for any query, but it would have a very adverse effect on query performance.
+=== Hit count accuracy
+You can limit the required accuracy of hit counts by setting `hit-count-accuracy` attribute.
+When dealing with large data sets, precise hit counts can impact performance.
+Setting a limit to the hit count accuracy, lets you achieve faster query responses while ensuring that the provided hit counts remain sufficiently accurate for your application's needs.
 
+The default accuracy of the `hit-count-accuracy` attribute is limited to `10000`.
+This means that for any query, {brandname} provides exact hit count up to maximum of 10 000.
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/topics/ref_query_ickle.adoc:62]: \10 000\10000\s ?

File: documentation/src/main/asciidoc/topics/ref_query_ickle.adoc [hunk 2]
```diff
@@ -48,24 +48,26 @@ You can change the default limit by setting the `query.default-max-results` cach
...
+If the effective hit count is higher than 10 000, {brandname} returns a lower bound estimate of the count.
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/topics/ref_query_ickle.adoc:63]: \10 000\10000\s ?

**Review by fax4ever — APPROVED** [review]
Great job @dvagnero.
Thanks

File: documentation/src/main/asciidoc/topics/ref_query_ickle.adoc (hunk 2 above)
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/ref_query_ickle.adoc]: Consistency! I went with 10,000 (that's what is recommended in the styleguide)

**Pull request merged**